src/codewhisperer/util/closingBracketUtil.ts

```diff
@@ -12,22 +12,54 @@ interface bracketMapType {
     [k: string]: string
 }
 
-const bracketMap: bracketMapType = {
+const quotes = ["'", '"', '`']
+
+const closing: bracketMapType = {
     ')': '(',
     ']': '[',
     '}': '{',
+    '>': '<',
+    "'": "'",
+    '"': '"',
+    '`': '`',
+}
+
+const openning: bracketMapType = {
+    '(': ')',
+    '[': ']',
+    '{': '}',
+    '<': '>',
+    "'": "'",
+    '"': '"',
+    '`': '`',
 }
 
-export const calculateBracketsLevel = (code: string, isRightContext: boolean = false) => {
+export const calculateBracketsLevel = (
+    editor: vscode.TextEditor,
+    code: string,
+    isRightContext: boolean = false,
+    end: vscode.Position,
+    start: vscode.Position
+) => {
     const bracketCounts: Map<string, number> = new Map([
         ['(', 0],
         ['[', 0],
         ['{', 0],
+        ['"', 0],
+        ["'", 0],
+        ['<', 0],
+        ['`', 0],
     ])
```

Comment:
Can the keys be derived from the `opening` map? Otherwise you need to make changes to both places.